akka-persistence-testkit/src/test/scala/akka/persistence/testkit/scaladsl/CommonSnapshotTests.scala

```diff
@@ -527,36 +527,6 @@ trait CommonSnapshotTests extends ScalaDslUtils {
 
     }
 
-    "test snapshot events with RetentionCriteria after sending commands" in {
```

Comment:
ok, we still have the other "test snapshot events with RetentionCriteria", which I guess is fine